auth_token/models.py

```diff
@@ -49,23 +47,18 @@ def save(self, *args, **kwargs):
              self.key = self._generate_unique_key()
         return super().save(*args, **kwargs)
 
-    def _generate_key(self):
-        """
-        Random ID generating
-        """
-        return force_text(binascii.hexlify(os.urandom(20)))
-
     def _generate_unique_key(self):
         """
         Generate random unique token key.
         """
-        key = self._generate_key()
+        from auth_token.utils import generate_key
```

Comment:
tam je někde křížový import?

Reply:
ano je. Radil bych spíše vyřešit ty křížové importy v utils

Reply:
right, fixe

Reply:
tohle už tedy můžeš odstranit